tests/suites/examples/example.sh

```diff
@@ -0,0 +1,40 @@
+run_example1() {
+    # Echo out to ensure nice output to the test suite.
+    echo
+
+    # The following ensures that a bootstrap juju exists
+    file="${TEST_DIR}/test-example1.txt"
+    ensure "example1" "${file}"
```

Comment:
from outside (I do know because you told me) it can be unclear what e.g. ensure actually does.
From context, one could guess creating a model. It would be cooler to be more explicit in the example.